cfgov/jinja2/v1/_includes/macros/filters.html

```diff
@@ -361,6 +364,23 @@
 
 {# A helper macro for filters()
    ========================================================================== #}
+{% macro _errors() %}
+    {% set messages = get_messages(request) %}
+    {%- if messages-%}
+        {% import 'macros/notification.html' as notification %}
+        <div class='error-container '>
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/606b7c1d369eb210b18ded6cb7f660ffbe3bf87e